src/styles/blocks/catalog.scss

```
@@ -17,9 +17,14 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition-duration: $animation-time;
```

It would be best if you explicitly named all properties that will change on transition, like:
```suggestion
    transition: property1 $animation-time, 
    	property2 $animation-time ;
```

---

Thank you, but there are only 2 changes to the animation; without them, the code will be shorter. But maybe I'm not right:)